hknweb/events/models.py

```diff
@@ -41,7 +41,13 @@ def __repr__(self):
 
     def __str__(self):
         return self.name
-
+    def admitted_set(self):
+        return self.rsvp_set.order_by("created_at")[:self.rsvp_limit]
+    
+    def waitlist_set(self):
```

Comment:
Also these two functions are not used anywhere. Maybe use them instead of duplicating the same code in views.py?